### PR TITLE
Analyze gene coverage

### PR DESCRIPTION
**Pull request by WilliamRichards2017**
Changing coverage thresholds reanalyzes coverage for all genes, while keeping a consistent selected gene and variant state

### CODE REVIEW

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://4d0a7221.quarantest.iobio.io

**Comment by WilliamRichards2017** [comment]
To test:

1)Run with demo data, and let the loading finish. 
2) Select  the access coverage from the nav bar, and set the minimum threshold to 100
3) The Assess Coverage insufficient coverage glyph on the nav bar should be properly updated to '5', and all the genes in the gene-panel should show the insufficient coverage glyph
4) The variant previously selected before assessing coverage should remain.

**Review by adityaekawade — APPROVED** [review]
Tested on quarantest as per the steps outlined above. Working as expected!

File: client/app/components/pages/GeneHome.vue [hunk 1]
```diff
@@ -1612,18 +1612,13 @@ export default {
...
-                self.promiseSelectFirstFlaggedVariant()
-                .then(function() {
-                  self.$refs.navRef.onShowVariantsTab();
-                })
-              }
-              else if(self.launchedFromClin){
-                self.promiseSelectFirstFlaggedVariant()
-                    .then(function() {
-                      self.$refs.navRef.onShowVariantsTab();
-                    })
```
Comment (adityaekawade) [inline client/app/components/pages/GeneHome.vue:1625]: @WilliamRichards2017  There is a check if `launchedFromClin` which is removed in this PR. Did you try this in clin to check if it did not have any side effects?

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://7af56278.quarantest.iobio.io

**Pull request merged**